src/brevitas/graph/base.py

```diff
@@ -191,6 +191,9 @@ def _init_new_module(self, old_module: Module, name=None):
     def _replace_old_module(self, model, old_module, new_module, load_state_dict=True):
         replace_module(model, old_module, new_module)
         if load_state_dict:
+            # work-around since weight_orig is from a previous quantization algorithm
+            if hasattr(old_module, 'weight_orig'):
+                new_module.register_buffer('weight_orig', old_module.weight_orig.data)
```

Comment:
I am playing with weight orig for other stuff and I believe it makes sense to move this to cpu and then move it to cuda on demand, otherwise we have total model duplication and it's not good.